MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -116,7 +116,15 @@ + (IMP)swizzleOriginalSelector:(SEL)originalSelector withCustomSelector:(SEL)cus
   IMP customImp = class_getMethodImplementation(self, customSelector);
   IMP originalImp = NULL;
   BOOL methodAdded = NO;
-  
+
+  // Replace implementation in super classes
+  Class baseClass = originalClass.superclass;
+  if( baseClass != [NSObject class]) {
```

Comment:
I agree the first comment that you need to check if it is implemented or not.
BTW for the second part, isn't NSObject always be the root class of other classes in Objective C? Just curios.